src/Transformation/Factory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Transformation;
```

Comment:
You mean we should nest that more deeply like `Transformation\FileTypes`? Would that mean from FileTypes to something else or from something to FileTypes? Do we need that now or could we also introduce that later when we see how this works out? In general this sounds like a good idea, though.